exec/cnex/lib/mmap_posix.c

```diff
@@ -0,0 +1,184 @@
+#include "mmap.h"
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "cell.h"
+#include "exec.h"
+#include "nstring.h"
+#include "number.h"
+#include "object.h"
+#include "stack.h"
+
+
+typedef struct tagTMmapObject {
```

Comment:
You should reconsider your use of `typedef` everywhere. It's just not necessary. All it does is (a) introduce all these odd `tagWhatever` names, and (b) save typing `struct` in some places. It's better to type `struct`.

Reply:
Removed typedef, and changed all references to MmapObject, instead of TMmapObject.